packages/react-sdk/src/components/ElementBar/DuplicateActiveElementButton/DuplicateActiveElementButton.test.tsx

```diff
@@ -290,3 +302,41 @@ describe('duplicate', () => {
     });
   });
 });
+
+describe('<DuplicateActiveElementButton />', () => {
+  let widgetApi: MockedWidgetApi;
+  let Wrapper: ComponentType<PropsWithChildren<{}>>;
```

Comment:
`deno lint` always tells me that `{}` isn't what I expect it to be.
I should use `Record<never, never>`.

Reply:
We don't use Deno. Our ESLint config has this excluded https://github.com/nordeck/matrix-neoboard/blob/main/eslint.config.mjs#L99

Reply:
I know we're not using Deno. Just pointing out that their linter taught me to not use `{}` anymore, because
> {} represents any non-nullish value, including literals like 0 and ""

Reply:
I do not disagree. That is how we have it currently coded into our linter rules. Can you please bring this to our daily discussion? We need to create a ticket and change the ESLint rules.

To bring this forward: type specified here.